ci/jenkins/mellanox/scripts/start_ci.sh

```diff
@@ -27,6 +27,8 @@ export ANTREA_CNI_REPO=${ANTREA_CNI_REPO:-'https://github.com/antrea-io/antrea.g
 export ANTREA_CNI_BRANCH=${ANTREA_CNI_BRANCH:-''}
 export ANTREA_CNI_PR=${ANTREA_CNI_PR:-''}
 export ANTREA_CNI_HARBOR_IMAGE=${ANTREA_CNI_HARBOR_IMAGE:-${HARBOR_REGISTRY}/${HARBOR_PROJECT}/antrea}
+export ANTREA_AGENT_CNI_HARBOR_IMAGE=${ANTREA_AGENT_CNI_HARBOR_IMAGE:-${HARBOR_REGISTRY}/${HARBOR_PROJECT}/antrea-agent}
+export ANTREA_CONTROLLER_CNI_HARBOR_IMAGE=${ANTREA_CONTROLLER_CNI_HARBOR_IMAGE:-${HARBOR_REGISTRY}/${HARBOR_PROJECT}/antrea-controller}
```

Comment:
I saw in workflows/build.yml you named them antrea-controller-ubuntu-amd64 and antrea-agent-ubuntu-amd64. Harbor will load the same images from dockerhub.